docs/examples.rst

```diff
@@ -24,6 +24,22 @@ is stored in-memory (in dictionaries)::
    # create project for managing Pastas data and models
    store = pst.PastaStore("my_project", conn)
 
+Using Pastas
+------------
+
+The following snippet shows how to use PastaStore with storage of 
+timeseries and models on disk as pas-files. This is the simplest implementation 
+that writes data to disk as no external dependencies are required::
+
+   import pastastore as pst
+
+   # define dict connector
```

Comment:
Is the `PasConnector` replacing the `DictConnector`? The comment here refers to the `DictConnector` and the code to the `PasConnector`